files/build_templates/sonic_debian_extension.j2

```diff
@@ -104,7 +104,7 @@ sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install $CONFIG
 sudo rm -rf $FILESYSTEM_ROOT/$CONFIG_ENGINE_WHEEL_NAME
 
 # Install Python client for Redis
-sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install "redis==2.10.6"
+sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install "redis==3.5.3"
```

Comment:
Just wanted to add a note here since I was going to ask: this redis client version supports both python2.7 and python3.

Reply:
Aaaaand you mentioned this in the swsssdk PR description already too. Doesnt hurt to double-check, I guess. :)